streaming/index.js

```diff
@@ -372,7 +372,7 @@ const startWorker = (workerId) => {
       }
 
       // Only send local-only statuses to logged-in users
-      if (payload.local_only && !req.accountId) {
+      if (payload.local_only && !req.accountId || !allowLocalOnly) {
```

Comment:
Parentheses would be good here - if I understand it correctly (&& higher preference than ||) then if allowLocalOnly is false, every status is refused here, even for a logged-in user?

I'm not sure I understand what the logic is supposed to be: if allowLocalOnly is true, then local_only messages should get through even if no accountId is available? if allowLocalOnly is *false* but there's an accountId available let the message through? Perhaps allowLocalOnly is a slightly confusing name?
```suggestion
      if (payload.local_only && !(req.accountId || allowLocalOnly)) {
```

Reply:
In this case, I am copying changes from glitch-soc to make this fix to our code base.  I believe you have the parentheses around the wrong set here.  It would have to be the left two items if you wanted to make it like that.